config/config.go

```diff
@@ -93,4 +95,7 @@ type Config interface {
 	// GetPeerBufferSize returns the size of the libhoney buffer to use for the peer forwarding
 	// libhoney client
 	GetPeerBufferSize() int
+
+	// GetSendTickerValue returns the duration to use to check for traces to send
+	GetSendTickerValue() time.Duration
```

Comment:
Propose switching all time-based configs to use TOML's time parsing so you would specify `1m30s` and it would show up as a duration. That would be a different PR though.